artillery/artillery

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+WARM_UP_DURATION=10
+RAMP_UP_DURATION=30
+SUSTAINED_LOAD_DURATION=300
+WARM_UP_VUSERS=1
+WARM_UP_TARGET_VUSERS=1
+RAMP_UP_TARGET_VUSERS=20
+SUSTAINED_LOAD_VUSERS=20
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --warm-up-duration)
+      WARM_UP_DURATION=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --ramp-up-duration)
+      RAMP_UP_DURATION=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --sustained-load-duration)
+      SUSTAINED_LOAD_DURATION=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --warm-up-vusers)
+      WARM_UP_VUSERS=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --warm-up-target-vusers)
+      WARM_UP_TARGET_VUSERS=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --ramp-up-target-vusers)
+      RAMP_UP_TARGET_VUSERS=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --sustained-load-vusers)
+      SUSTAINED_LOAD_VUSERS=$2
+      shift # past argument
+      shift # past value
+      ;;
+  esac
```

Comment:
In a future PR might be nice to have a default *) here to catch misspelled parameters and bail out.